apps/aecontract/src/aect_sophia.erl

```diff
@@ -27,13 +28,8 @@
 
 -export_type([ wrapped_code/0 ]).
 
--define(SOPHIA_CONTRACT_VSN_1, 1).
--define(SOPHIA_CONTRACT_VSN_2, 2).
-
 %% After hard fork Minerva, we accept ?SOPHIA_CONTRACT_VSN_2 in serialization
-%% Therefore, switch the present version to ?SOPHIA_CONTRACT_VSN_2 when release
-%% after hard fork.
--define(SOPHIA_CONTRACT_VSN, ?SOPHIA_CONTRACT_VSN_1).
+-define(SOPHIA_CONTRACT_VSN, ?SOPHIA_CONTRACT_VSN_2).
```

Comment:
This macro is used in `serialize/1`, keeping old value can be misleading for the reader.